Documentation.app/Contents/MacOS/Composition.playground/Pages/Composing functions.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,103 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: Composing functions
+ */
+// nef:end
+// nef:begin:hidden
+import Bow
```

Comment:
I think you need to import Foundation to use `Date`...

```suggestion
import Foundation
import Bow
```

Reply:
It seems it is not necessary.